portfolio/src/main/webapp/visualizer.js

```diff
@@ -0,0 +1,30 @@
+google.charts.load('current', {
+  'packages':['geochart'],
+  'mapsApiKey': 'AIzaSyCsIDmQAVTMjvXr6NZBm1O8b9Ve9EWREwk'
+});
+google.charts.setOnLoadCallback(drawRegionsMap);
+
+function drawRegionsMap() {
+  fetch('/covid-data').then(response => response.json()).then((covidData) => {
+
+    console.log(covidData);  
+
+    /*const data = new google.visualization.DataTable();
```

Comment:
Is the console.log and commented out section intentional? If it is that's okay for this project but I'd add a comment above it.